android/src/main/java/com/swmansion/rnscreens/SearchBarView.kt

```diff
@@ -144,6 +144,8 @@ class SearchBarView(reactContext: ReactContext?) : ReactViewGroup(reactContext)
         text?.let { screenStackFragment?.searchView?.setText(it) }
     }
 
+    private val surfaceId = UIManagerHelper.getSurfaceId(this)
+
```

Comment:
I wonder whether I can cache this value or not, but this computation should not be expensive.

Reply:
I think we should not cache it since in the future `surfaceId` should change based on the part of the application you are in. For now noone does it, but we won't have to change it later if we do it correctly right now 🤷‍♂️